packages/safe-core-sdk/tests/utils/setupEthAdapter.ts

```diff
@@ -9,7 +9,7 @@ export async function getEthAdapter(signer: Signer): Promise<EthAdapter> {
   switch (process.env.ETH_LIB) {
     case 'web3':
       const signerAddress = await signer.getAddress()
-      const web3AdapterConfig: Web3AdapterConfig = { web3, signerAddress }
+      const web3AdapterConfig: Web3AdapterConfig = { web3: web3 as any, signerAddress }
```

Comment:
This is needed. `web3` instance is coming from `hardhat` package (not `web3`) and tests would complain telling that "is not assignable".